src/Symfony/Component/HttpKernel/Controller/ArgumentResolver/QueryParameterValueResolver.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\HttpKernel\Controller\ArgumentResolver;
+
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpKernel\Attribute\MapQueryParameter;
+use Symfony\Component\HttpKernel\Controller\ValueResolverInterface;
+use Symfony\Component\HttpKernel\ControllerMetadata\ArgumentMetadata;
+use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
+
+final class QueryParameterValueResolver implements ValueResolverInterface
```

Comment:
But why? It will ease navigation between classes - `MapQueryParameter`, `MapQueryParameterValueResolver`, `MapQueryParameterValueResolverTest`

Reply:
Looking at existing resolvers, we don't prefix them right now. It would also stop IDEs from autosuggesting resolvers instead of attributes when typing "Map"